Source/Orts.Formats.Msts/CabViewFile.cs

```diff
@@ -446,6 +446,13 @@ protected virtual float ParseSwitchVal(STFReader stf)
             stf.SkipRestOfBlock();
             return switchVal;
         }
+        protected virtual float ParseRotation(STFReader stf)
+        {
+            stf.MustMatch("(");
+            var rotation = -MathHelper.ToRadians((float)stf.ReadDouble(0));
```

Comment:
The angle is inverted here, which means the rotation is clockwise instead of counterclockwise. Was this intentional?

Reply:
Hi Ryan,
this is coherent with what is already included in OR for the rotation of digital controls. And the rotation is counterclockwise for positive angles, at least for vertical bars, as you can see on the white gauge at the left of the attached picture http://www.interazioni-educative.it/Varie/GaugeRotation.jpg , where ORTSAngle has been set to 12. So, yes, the minus sign is intentional

Reply:
Ok, that makes sense. Thanks for clarifying!